crates/biome_js_analyze/src/lint/a11y/no_aria_hidden_on_focusable.rs

```diff
@@ -52,6 +59,40 @@ declare_lint_rule! {
     }
 }
 
+declare_node_union! {
```

Comment:
This implementation is copy-pasted from https://github.com/biomejs/biome/blob/8d29726acb4596b5abcfae1ddb48cdf2d1bcbe86/crates/biome_js_analyze/src/lint/a11y/no_noninteractive_tabindex.rs#L61.

I'm not certain if that warrants breaking it out into some shared module or if refactoring in a separate PR is more preferable. Happy to do what maintainers think is best!

Reply:
We usually move shared code whenever it's possible. For your case, you'll have to update `biome_js_syntax/src/expr_ext.rs`, copy-paste the new node and expose it publicly.

Reply:
Sounds great! Done!